src/elements/TabbedContainer/TabbedContainer.js

```diff
@@ -0,0 +1,99 @@
+import './TabbedContainer.less';
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import Div from '../Div/Div';
+import List from '../List/List';
+
+class TabbedContainer extends Component {
+	constructor(props) {
+		super(props);
+
+		this.state = {
+			// set default showing tab (0 = first tab, 1 = second tab, etc...)
+			activeIndex: 0
+		};
+
+		this.changeTabs = this.changeTabs.bind(this);
+		this.renderNavItems = this.renderNavItems.bind(this);
+		this.renderOneNavItem = this.renderOneNavItem.bind(this);
+	}
+
+	changeTabs(event, key) {
+		event.preventDefault();
+
+		this.setState({
+			activeIndex: key
+		});
+	}
+
+	renderNavItems(key) {
+		const tab = this.props.children[key] ? this.props.children[key] : null,
+			keyNum = Number(key);
+
+		const navItemClasses = classNames(
+			this.state.activeIndex === keyNum ? 'active' : ''
+		);
+
+		return (
+			<li key={key} className={navItemClasses}>
+				<a href="#" onClick={ (e) => this.changeTabs(e, keyNum) }>{ tab.props.title }</a>
+			</li>
+		);
+	}
+
+	renderOneNavItem() {
+		const oneTab = React.Children.only(this.props.children);
+
+		return (
+			<li className="active">
+				<a href="#">{ oneTab.props.title }</a>
+			</li>
+		);
+	}
+
+	render() {
```

Comment:
In the `render()` function, it's typically best practice to destructure your state and props at the top and use those instead of littering the code with `this.state...` and `this.props...`. It's not too bad since you're using 1 of each in this instance, but with any more you'd want to do something like this:

```js
const { activeIndex } = this.state,
    { children } = this.props;

// use em up ....
```

Reply:
This is also something we should include in our style guide/linter. I'm in favor of _always_ destructuring because it makes it easy to remember across all the devs, and makes small refactors simple (since switching from state to props for the data is contained to a single line)